Doc/whatsnew/3.13.rst

```diff
@@ -509,19 +509,6 @@ Incremental garbage collection
 The cycle garbage collector is now incremental.
```

Comment:
shouldn't this entire section from line 504 through the gh-108632 credit below be removed?